Jenkinsfile

```
@@ -0,0 +1,23 @@
+pipeline {
+  agent {
+    label "kubernetes-qat-envoy"
+  }
+  environment {
+    QAT_VERSION="1.7.l.4.5.0-00034"
+    QAT_SHA_256="c42a3afc1a5c76d441eaca8b97dc1f9ee64939ec001539ee1a2f3b39b7543c8e"
+    REPO_URL="https://github.com/intel/kubernetes-qat-envoy"
+  }
+  stages {
+    stage('Create container for QAT-accelerated Envoy') {
+      options {
+        timeout(time: 90, unit: "MINUTES")
+        retry(2)
+      }
+      steps {
+        sh "wget https://01.org/sites/default/files/downloads/qat${QAT_VERSION}.tar.gz"
+        sh "sha256sum qat${QAT_VERSION}.tar.gz | grep $QAT_SHA_256"
+        sh "docker image build -t envoy-qat:devel -f Dockerfile.envoy ."
```

@chuyd nice patch!.  If #15 lands first, it will simplify the things, only this line will be needed

---

BTW, the docker build instruction should include some [additional parameters](https://github.com/intel/kubernetes-qat-envoy/blob/master/vagrant/configure-qat-envoy.yml#L67)

```suggestion
        sh "docker image build -t envoy-qat:devel --no-cache --pull -f Dockerfile.envoy ."
```

---

Does Jenkins have any built-in support for docker commands? It feels to me that caching could be smarter than just a on/off toggle.

---

Since #15 is now merged, you could simplify the file as suggested by @devimc to make use of the refactored build scripts.